File: Java/VSCode/src/FizzBuzzConEntrada.java

import java.util.Scanner;


public class FizzBuzzConEntrada {
    
    private static int n;

    public static void main (String[] args){
        try (Scanner sc = new Scanner(System.in)) {
            System.out.println("Introduce un número: ");
            n = sc.nextInt();
        }
        for (int i = 1; i <= n; i++){
            if (i % 3 == 0 && i % 5 == 0){
                System.out.println("FizzBuzz");
            } else if (i % 3 == 0){
                System.out.println("Fizz");
            } else if (i % 5 == 0){
                System.out.println("Buzz");
            } else {
                System.out.println(i);
            }
        }
    }
}

File: Python/FizzBuzzConEntrada.py
"""
Instrucciones:
Versión modificada de FizzBuzz que:
- Solicita al usuario un número n
- Imprime la secuencia FizzBuzz hasta ese número
- Sigue las mismas reglas que FizzBuzz tradicional
"""

def main():
    try:
        print("Introduce un número: ")
        n = int(input())
        
        for i in range(1, n + 1):
            if i % 3 == 0 and i % 5 == 0:
                print("FizzBuzz")
            elif i % 3 == 0:
                print("Fizz")
            elif i % 5 == 0:
                print("Buzz")
            else:
                print(i)
    except ValueError:
        print("Por favor, ingrese un número válido")

if __name__ == "__main__":
    main()